### DISCUSSION TITLE
Is there a v0.18.0 release of SQLPage (referenced in docs)

### DISCUSSION THREAD

**Discussion by shah** [sqlpage/SQLPage#177] [Q&A] [answered]
The documentation for SQLPage refers to some components (e.g. `breadcrumb`) as being available in `v0.18` but I do not see that release. Is there a v0.18.x release of SQLPage?

**Comment by lovasoa** [comment, marked as answer]
Hello ! v0.18 is the next release, it is not finished yet, but you can already try it with `lovasoa/sqlpage:main` on docker if you are impatient :)

**Reply by shah** [reply]
Awesome, glad to see I did not miss anything -- I'll wait for the binary release. Thanks!
